src/jira/client/index.ts

```diff
@@ -153,13 +153,22 @@ async function getJiraClient(
 							}
 						}
 					),
-				delete: (gitHubInstallationId: string | number) =>
-					instance.delete(`/rest/devinfo/0.10/bulkByProperties`,
-						{
-							params: {
-								installationId: gitHubInstallationId
-							}
-						})
+				delete: async (gitHubInstallationId: string | number) =>
+					Promise.all([
+
+						// We are sending devinfo events with the property "installationId", so we delete by this property.
+						instance.delete(
+							`/rest/devinfo/0.10/bulkByProperties?installationId=${gitHubInstallationId}`
```

Comment:
don't use string templating for URL. Use the url as a base, but all variable needs to go either through `params` or `urlParams` (in this case the former)